### ISSUE TITLE
real time metrics visualization

### ISSUE THREAD

**Issue by Qazalbash** [gwkokab/gwkokab#654]
Add a feature related to real time metric visualization through [tensorboard](https://www.tensorflow.org/tensorboard). Metrics can be, training loss, training/production chains etc. We will first provide this feature for analyses using flowMC.

**Comment by smSafiHaider** [comment]
Hey, bro i am willing to contribute to this, can you provide me guidance?

**Comment by Qazalbash** [comment]
Tensorboard will be integrated in these lines.

https://github.com/gwkokab/gwkokab/blob/383146515804b45ba9cdd98470327ee0963f2cae/src/kokab/core/flowMC_based.py#L426-L461

You have to show these quantities on appropriate graph.

https://github.com/gwkokab/gwkokab/blob/383146515804b45ba9cdd98470327ee0963f2cae/src/kokab/core/flowMC_based.py#L537-L548

**Comment by Qazalbash** [comment]
#670 has implemented the same feature requested in this issue, by using pre-existing APIs. So, I'm closing this issue.

Thank you, @smSafiHaider, for taking an interest in it.
